nbbuild/cluster.properties

```diff
@@ -445,8 +445,10 @@ nb.cluster.ide=\
         o.eclipse.mylyn.wikitext.core,\
         o.eclipse.mylyn.wikitext.markdown.core,\
         o.eclipse.mylyn.wikitext.textile.core,\
+        o.jboss.spec.javax.servlet,\
         o.n.swing.dirchooser,\
         o.openidex.util,\
+        o.slf4j.api,\
```

Comment:
... or this?

Reply:
JGit 4.11.0 has an OSGi dependency on the SLF4J API 1.7.2, and the only package I found on Maven Central that provided this requirement is the one from Apache Directory Studio!

Click here for details: 

https://github.com/eclipse/jgit/blob/61e4f1665221626cfd31a5826bff527ef7ce5719/pom.xml#L214
https://github.com/eclipse/jgit/blob/61e4f1665221626cfd31a5826bff527ef7ce5719/org.eclipse.jgit/META-INF/MANIFEST.MF#L155

Reply:
SLF4J is under MIT license and I don't see that mentioned in the patch https://www.slf4j.org/license.html

I wonder, is adding just the API sufficient? Don't we need to add an adaptor for java.util.logging? 

Is logging even enabled for jgit?

Reply:
You're right! I've used the Apache license because the version I've chosen is coming from the Apache Directory Studio project but I guess I should have provided both...